frame/staking/src/mock.rs

```diff
@@ -289,7 +289,7 @@ impl ExtBuilder {
 	pub fn build(self) -> sp_io::TestExternalities {
 		self.set_associated_consts();
 		let mut storage = system::GenesisConfig::default().build_storage::<Test>().unwrap();
-		let balance_factor = if self.existential_deposit > 0 { 256 } else { 1 };
+		let balance_factor = if self.existential_deposit > 1 { 256 } else { 1 };
```

Comment:
These two lines also have changed in substrate(they are 0 before):

https://github.com/paritytech/substrate/blame/v2.0.0-alpha.3/frame/staking/src/mock.rs#L290